pages/portfolio.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Angela Portfolio</title>
+  <link href="../normalize.css" rel="stylesheet"/>
+  <link href="../styles/styles.css" rel="stylesheet"/>
+  <link href="../styles/portfolio-styles.css" rel="stylesheet"/>
+  <link href="https://fonts.googleapis.com/css?family=Pacifico" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Poiret+One" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <h1>Portfolio</h1>
+  </header>
+  <nav>
+    <ul class="nav-links">
+      <li><a href="../index.html">Home</a></li>
+      <li><a href="../pages/blog.html">Blog</a></li>
+      <li><a href="../pages/about.html">About</a></li>
+    </ul>
+  </nav>
+
+  <section class="ada-education">
+    <h2> This year I'll be:</h2>
+    <p> Rapidly learning and implementing full stack technologies, including Ruby, Rails, Javascript, HTML, and CSS, developing and consuming RESTful APIs, and studying database and computer science fundamentals</p>
+  </section>
+
+  <section class="project-container">
+    <h2>Check out some of my recent projects:</h2>
+    <ul class="projects">
+      <li class="a"><a href="https://github.com/AngelaPoland/hotel">Hotel</a></li>
```

Comment:
Maybe some rollover effects would make it very clear these are links.  

Also this isn't so good on smaller screens.